modules/tensor_mechanics/src/materials/ComputeFiniteStrain.C

```diff
@@ -11,6 +11,7 @@ InputParameters validParams<ComputeFiniteStrain>()
 {
   InputParameters params = validParams<ComputeStrainBase>();
   params.addClassDescription("Compute a strain increment and rotation increment for finite strains.");
+  params.set<bool>("stateful_displacements") = true;
```

Comment:
Ugh... really? So, is set deprecated or what? I really prefer the set otherwise as it is much more expressive. The re-adding is confusing IMO, as users might look at the code, see a "new" parameter which is never fetched anywhere in the class or the derived classes.